docs/tfengine/schemas/iam_members.md

```diff
@@ -7,3 +7,7 @@
 | Property | Description | Type | Required | Default | Pattern |
 | -------- | ----------- | ---- | -------- | ------- | ------- |
 | iam_members | [Module](https://github.com/terraform-google-modules/terraform-google-iam) | object | false | - | - |
+| iam_members.*pattern* | - | array(object) | false | - | ^storage_bucket\|project\|organization\|folder$ |
+| iam_members.*pattern*.bindings | Map of IAM role to list of members to grant access to the role. | object | false | - | - |
+| iam_members.*pattern*.bindings.*pattern* | - | array(string) | false | - | .+ |
```

Comment:
Just an extra comment before you merge.

Look at what happened to this line

![Captura de Pantalla 2021-07-20 a la(s) 10 55 36](https://user-images.githubusercontent.com/33379285/126356376-7461a7bd-6952-42ad-966d-9fefd627e87f.png)

Seems like we're loosing information

Reply:
Not really. The line immediately above it has the pattern for the element right after iam_members, and this line has the pattern for the element right after bindings.

Reply:
Gotcha! I didn't put enough attention to see that it had been previously defined.

Thanks!